x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/step_define_summary.tsx

```diff
@@ -132,23 +132,21 @@ export const StepDefineSummary: FC<Props> = ({
         </EuiFormRow>
 
         <EuiSpacer size="m" />
-        <EuiText>
-          <DataGrid
-            {...pivotPreviewProps}
-            copyToClipboard={getPivotPreviewDevConsoleStatement(previewRequest)}
-            copyToClipboardDescription={i18n.translate(
-              'xpack.transform.pivotPreview.copyClipboardTooltip',
-              {
-                defaultMessage: 'Copy Dev Console statement of the pivot preview to the clipboard.',
-              }
-            )}
-            dataTestSubj="transformPivotPreview"
-            title={i18n.translate('xpack.transform.pivotPreview.PivotPreviewTitle', {
-              defaultMessage: 'Transform pivot preview',
-            })}
-            toastNotifications={toastNotifications}
-          />
-        </EuiText>
+        <DataGrid
+          {...pivotPreviewProps}
+          copyToClipboard={getPivotPreviewDevConsoleStatement(previewRequest)}
+          copyToClipboardDescription={i18n.translate(
+            'xpack.transform.pivotPreview.copyClipboardTooltip',
+            {
+              defaultMessage: 'Copy Dev Console statement of the pivot preview to the clipboard.',
+            }
+          )}
+          dataTestSubj="transformPivotPreview"
+          title={i18n.translate('xpack.transform.pivotPreview.PivotPreviewTitle', {
```

Comment:
not relevant to this PR at all, but the first time I see a translation key starting with a capital letter 👀